tensorflow/contrib/lookup/lookup_ops.py

```diff
@@ -357,24 +358,26 @@ def __init__(self,
           use_node_name_sharing=use_node_name_sharing,
           key_dtype=key_dtype,
           value_dtype=value_dtype,
-          name=name)
+          name=self._name)
     else:
       self._table_ref = gen_lookup_ops.mutable_hash_table_of_tensors_v2(
           shared_name=shared_name,
           use_node_name_sharing=use_node_name_sharing,
           key_dtype=key_dtype,
           value_dtype=value_dtype,
           value_shape=self._default_value.get_shape(),
-          name=name)
+          name=self._name)
+
     if executing_eagerly:
```

Comment:
Can we replace the op_name (this block) with your unique name now?

I'd suggest keeping it as a method-local variable until it gets passed to super().__init__, since it looks like that sets self._name (and it would be unfortunate to have multiple names).